nncf/torch/extensions/__init__.py

```diff
@@ -20,7 +20,7 @@ class ExtensionsType(enum.Enum):
 def get_build_directory_for_extension(name: str) -> Path:
     build_dir = Path(_get_build_directory('nncf/' + name, verbose=False)) / torch.__version__
     if not build_dir.exists():
-        nncf_logger.debug("Creating build directory: {}".format(str(build_dir)))
+        nncf_logger.debug(f"Creating build directory: {str(build_dir)}")
```

Comment:
That's minor in general. Wouldn't it be better to use `.as_posix()` than `str()`?